app/views/shopping/_recipe_food_row.html.erb

```diff
@@ -0,0 +1,8 @@
+<% qty = RecipeFood.joins(:food).where(food_id: food.id).sum('quantity') # There has to be a better way to do this %>
+<% if qty > 0 %>
+  <tr>
+    <td><%= food.name %></td>
+    <td><%= qty %></td>
+    <td><%= food.price * qty %></td>
+  </tr>
+<% end %>
```

Comment:
**Optional Changes**
- [ ] Logic belongs to the view helper file. Again, you want to follow the `KISS` principle here.

Reply:
I'm not sure about how to better refactor this. It's conditional rendering, which seems appropriate in a view. I could go with a view helper, but the fact that I'm moving such a small condition to a helper method it's not very `KISS` in my opinion.